config/default/ActivityMonitoringService.conf.php

```diff
@@ -10,4 +10,5 @@
 return new ActivityMonitoringService([
     ActivityMonitoringService::OPTION_ACTIVE_USER_THRESHOLD => 300,
     ActivityMonitoringService::OPTION_COMPLETED_ASSESSMENTS_AUTO_REFRESH => 30,
+    ActivityMonitoringService::AUTO_REFRESH_INTERVAL => 60,
```

Comment:
Having these two properties may be confusing. I would suggest to combine them in one.